### PR TITLE
Fix popup order selection

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: #12694

- Respect the order in the list for the popup instead of the order we checked.

popup-order

For acceptance:

1. Create a new map
2. Add a new layer. The Dataset should be more than one field
3. Go to PopUp tab
4. Click on Hover tab
5. In the field's list select some random fields
6. Check that the order in the popup matches with the list

### CODE REVIEW

**Comment by piensaenpixel** [comment]
Ohhhh <3

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js [hunk 1]
```diff
@@ -69,15 +69,13 @@ module.exports = CoreView.extend({
...
-
     var selectedAll = true;
 
     _.each(this._getColumnNames(), function (field) {
-      if (!self.model.containsField(field)) {
+      if (!this.model.containsField(field)) {
         selectedAll = false;
       }
-    });
+    }.bind(this));
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: _.each admits a third parameter with the context, let's use it here

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js [hunk 2]
```diff
@@ -91,25 +89,23 @@ module.exports = CoreView.extend({
...
     var promises = [];
 
     _.each(names, function (f) {
-      if (!self.model.containsField(f)) {
-        promises.push(self.model._addField(f));
+      if (!this.model.containsField(f)) {
+        promises.push(this.model._addField(f));
       }
-    });
+    }.bind(this));
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: _ context as well

**Comment by nobuti** [comment]
👌

**Pull request merged**